sale_order_amount_history/init_hooks.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID, fields
+from odoo.api import Environment
+
+
+def post_init_hook(cr, registry):
+    env = Environment(cr, SUPERUSER_ID, {})
+    orders = env["sale.order"].search([])
+    for order in orders:
```

Comment:
Is this code means only super user can look history_vals?

Reply:
No, it means that the operation is done with the superuser so that it's not blocked by the access rights.

Reply:
Thank you, now I understand to login non-admin user.